larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -49,53 +49,54 @@ void* Init(TfLiteContext* context, const char* buffer, std::size_t length) {
   const std::uint8_t* buffer_t = reinterpret_cast<const std::uint8_t*>(buffer);
   const flexbuffers::Map& m = flexbuffers::GetRoot(buffer_t, length).AsMap();
 
-  // reading the op's input arguments into the "conv_params" struct
+  // Read the op's input arguments into the "conv_params" struct
+
   LCE_ENSURE_PARAM(conv_params, context, !m["stride_height"].IsNull());
   LCE_ENSURE_PARAM(conv_params, context, !m["stride_width"].IsNull());
   LCE_ENSURE_PARAM(conv_params, context, !m["dilation_height_factor"].IsNull());
   LCE_ENSURE_PARAM(conv_params, context, !m["dilation_width_factor"].IsNull());
   LCE_ENSURE_PARAM(conv_params, context, !m["padding"].IsNull());
+  LCE_ENSURE_PARAM(conv_params, context, !m["pad_values"].IsNull());
+  LCE_ENSURE_PARAM(conv_params, context, !m["channels_in"].IsNull());
```

Comment:
The 0.4 release contained breaking op definition changes meaning that we no longer have to deal with the possibility of these attributes not being defined.